src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

```diff
@@ -564,7 +566,7 @@ function MobileView(props: CodingBountiesProps) {
                             leadingImageContainerStyle={{
                               left: 450
                             }}
-                            disabled={!props?.deleteAction}
```

Comment:
we should probably also keep this logic aswell since if there is no deleteAction the button should not be enabled

Reply:
Hey Kev, did you see the video? With the logic implemented the button works fine

https://github.com/stakwork/sphinx-tribes-frontend/assets/34518489/0796a8ba-a3f3-495a-915f-a71ec1e478f0

Reply:
Yes I know it works, but if someone were to reuse this component elsewhere and not include a deleteAction then the button would not be disabled. So we should check if a deleteAction is passed as a prop otherwise disable the delete button